lib/pools/src/utils/sql_query_macros.rs

```diff
@@ -218,6 +218,7 @@ macro_rules! __sql_query_as_raw {
 		)*
 		.$action($crdb)
     };
+	// TODO: This doesn't work with `fetch`
```

Comment:
Why doesn't this work? We should just remove it if it's broken.

Reply:
Its effectively trying to do `sqlx::fetch().await` which doesn't compile since it returns a stream, not a future